src/ServiceInsight/Saga/SagaWindowViewModel.cs

```diff
@@ -99,6 +106,11 @@ string GetMajorVersion(StoredMessage message)
             return version?.Split('.').First();
         }
 
+        void ClearSaga()
+        {
+            Data = null;
+        }
+
```

Comment:
Fair enough, perhaps it does indeed have some value. It may just be a matter of taste. I'm OK to have the separate method.